config/betterquesting/DefaultQuests/Quests/Tier0StoneAge-AAAAAAAAAAAAAAAAAAAAAQ==/YourFirstTool-AAAAAAAAAAAAAAAAAAAB2A==.json

```diff
@@ -62,7 +62,7 @@
-      "desc:8": "When your tinkers tables are ready you can craft your first §oreal§r tool. If you didn\u0027t find a village yet (to grab some parts from the chest), you can only craft flint tools.\n\nLet\u0027s start with a pickaxe, an axe and a shovel. At this point swords are unobtainable since they need metals like iron or bronze which you will get at a later stage. In the meantime use your gregtech swords.\n\nYour new tools are weak at first, but the more you use them the better they will get. You get a modifer slot after some levels, needing more as you level it up. They will level up to level 99 if you play long enough, with 20 modifier slots. Tool parts can be replaced for a bit of level XP, however the tool must be fully repaired first.\n\nYou need to level up the mining level each time you replace the parts to unleash the full power of the pickaxe. Stone-\u003eCopper for this pickaxe. If you manage to get a Zombie, Skeleton, or Creeper head, you can use it to skip the level-up process. This doesn\u0027t use up a modifier level, but does change the look until you change the pickaxe part out.\n\nItem\u003dUsable when mining level is X or less\nZombie\u003d1\nSkeleton\u003d2\nCreeper\u003d4\nWither Skeleton\u003d7\nNether Star\u003d8\n\n§3You can use paper binding to add extra modifier, but not very much important as tool level exists."
+      "desc:8": "When your tinkers tables are ready, you can craft your first §oreal§r tool. If you didn\u0027t find a village yet to grab some parts from the chest, you can only craft flint tools.\n\nLet\u0027s start with a pickaxe, an axe, and a shovel. At this point swords are unobtainable since they need metals like iron or bronze which you will get at a later stage. In the meantime use your gregtech swords.\n\nKeep in mind that most of the original tool parts from Tinker\u0027s Construct are not shown in NEI. That doesn\u0027t mean that you can\u0027t craft them - use the tooltips on patterns and molds as guidance. \n\nYour new tools are weak at first, but the more you use them the better they will get. You get a modifier slot after some levels, needing more as you level it up. They will level up to level 99 if you play long enough, with 20 modifier slots. Tool parts can be replaced for a bit of level XP, however the tool must be fully repaired first.\n\nYou need to level up the mining level each time you replace the parts to unleash the full power of the pickaxe. Stone-\u003eCopper for this pickaxe. If you manage to get a Zombie, Skeleton, or Creeper head, you can use it to skip the level-up process. This doesn\u0027t use up a modifier level, but does change the look until you change the pickaxe part out.\n\nItem\u003dUsable when mining level is X or less\nZombie\u003d1\nSkeleton\u003d2\nCreeper\u003d4\nWither Skeleton\u003d7\nNether Star\u003d8\n\nYou can use paper binding to add an extra modifier, but it\u0027s very important because you will get modifiers with tool levels."
```

Comment:
"Keep in mind that most of the original tool parts from Tinker\u0027s Construct are not shown in NEI. That doesn\u0027t mean that you can\u0027t craft them - use the tooltips on patterns and molds as guidance."

if you find a bug like this you should make a ticket not change the quest. remove here.

Reply:
There is an existing issue that seems to cover this https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/9920